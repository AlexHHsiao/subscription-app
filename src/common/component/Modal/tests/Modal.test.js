import React from 'react';
import {render} from '@testing-library/react';
import ModalComponent from '../Modal';

const mockData = {
    open: true,
    onClose: () => {},
    modalData: {
        message: '',
        type: 'Error',
        title: 'Something went wrong',
    }
};

test('<ModalComponent/>', () => {
    const {container: {firstChild}} = render(<ModalComponent {...mockData}/>);
    expect(firstChild).toMatchSnapshot();
});
