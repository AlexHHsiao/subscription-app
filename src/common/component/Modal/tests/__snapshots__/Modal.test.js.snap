// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ModalComponent/> 1`] = `null`;
